New topic - Fear

LooseEthics opened the pull request:
Monika talks about scary stuff.
There are two things which I'd like to get some feedback on.
The first is how this topic shows up. It's supposed to get queued into Monika's random chatter, appear once on its own, and only be accessible from the repeat menu afterwards. I'm not sure if I set that up properly.
The second is the expressions. I tried to match them to the lines as best as I could, but some still feel a bit off to me.
The attachment is the original draft file with comments, alternative ideas,
 and whatnot. Just in case someone wants to read that.
[worst_fear.txt](https://github.com/Monika-After-Story/MonikaModDev/files/4634518/worst_fear.txt)

jmwall24 commented:
Is there a particular reason you only want it to be random the one time?

jmwall24 commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -14435,3 +14435,39 @@ label monika_language_nuances:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel='monika_fear',
```
the event label and the label need to match

jmwall24 commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -14435,3 +14435,39 @@ label monika_language_nuances:
+
+label monika_worst_fear:
+    m 3eud "Say, [player]..."
+    m 1eud "This is kind of a weird question, but... {w=0.5} Is there something you're afraid of?"
```
```suggestion
    m 1eud "This is kind of a weird question, but...{w=0.5}is there something you're afraid of?"
```
Since this should all be one sentence, no space

jmwall24 commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -14435,3 +14435,39 @@ label monika_language_nuances:
+    m 1eub "I don't mean the everyday, mundane kind of fear, like spilling a drink and ruining your favorite shirt."
+    m 1euc "I mean, like, the kind of deep-seated fear that fills you with dread when you even think about it."
+    m 1ruc "For me, losing you would obviously be on the top of {i}that{/i} particular list."
+    m 1ekd "I told you before, didn't I? {w=0.5} I don't know what I'd do if something happened to you."
```
```suggestion
    m 1ekd "I told you before, didn't I? {w=0.5}I don't know what I'd do if something happened to you."
```

jmwall24 commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -14435,3 +14435,39 @@ label monika_language_nuances:
+    m 3eua "But as long as we're being hypothetical..."
+    m 2esc "What truly terrifies me is the thought that none of this is real."
+    m "That one day I'll wake up, back in my bed, and realize it was all just a dream."
+    m "That there is no game,{w=0.2} no epiphany,{w=0.2} no... {w=0.5}{nw}"
```
```suggestion
    m "That there is no game,{w=0.2} no epiphany,{w=0.2} no...{w=0.5}{nw}"
```

jmwall24 commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -14435,3 +14435,39 @@ label monika_language_nuances:
+    m 2fkc "That you, your love, perhaps even the entirety of the literature club were all nothing but a figment of my own imagination."
+    m "That my old world, as grey and meaningless as it now seems, is all there truly is."
+    m 2lkc "..."
+    m 1rksdlb "Ahaha~{nw}"
```
```suggestion
    m 1rksdlb "Ahaha~ {nw}"
```

jmwall24 commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -14435,3 +14435,39 @@ label monika_language_nuances:
+    m 2lkc "..."
+    m 1rksdlb "Ahaha~{nw}"
+    extend 1eksdlb "Sorry, that got pretty dark, didn't it?"
+    m 1eksdla "I feel kinda silly now.{w=0.5}{nw}"
```
```suggestion
    m 1eksdla "I feel kinda silly now.{w=0.5} {nw}"
```

LooseEthics commented:
I just thought that, with the way it's written, it might be a bit strange for this to be a reoccurring topic.

multimokia commented:
> I just thought that, with the way it's written, it might be a bit strange for this to be a reoccurring topic.

To me, it flows similarly to the flowers for Algernon topic, which is not derandomed. That topic works fine, and I don't really see anything here which needs it to derandom.

jmwall24 commented:
@LooseEthics could you email me when you get a chance? email is publicly available on my profile. Thanks.

LooseEthics commented:
Right, so after actually setting up my testing install and reading this in mas, I see what you mean. So the derandom's gone.
New expressions get implemented on public updates, correct? 2fkc isn't currently visible.

LooseEthics commented:
Added the missing expressions. They show up fine on my end, but it might be better if someone double-checked they're implemented correctly.
Also added a wait on one of the lines because having Monika move her eyes so fast was unnatural.

jmwall24 commented:
I'll get to this this week
